templates/api.dot

```diff
@@ -34,14 +34,20 @@ import Json.Decode.Pipeline as JDP
 {{~ it.operations :op }}
 
 {-| {{= op.doc }}
+{{? op.input && op.input.members }}
+__Parameters__
+
+{{~ op.input.members :m }}* `{{= m.key }}` __:__ `{{= m.value.type }}`
+{{~}}
+{{?}}
```

Comment:
This extra documentation helps when there are a bunch of basic type (ex, `String`, `Int`) parameters